root.tf

```diff
@@ -315,7 +320,7 @@ module "file_format_sqs_queue" {
   function                 = "file-format"
   dead_letter_queue        = module.backend_check_failure_sqs_queue.sqs_arn
   redrive_maximum_receives = 3
-  visibility_timeout       = 180
+  visibility_timeout       = 900
```

Comment:
What's the reason for configuring a longer visibility timeout on this queue compared to the checksum one?

Reply:
The queue timeout needs to be at least as long as the lambda timeout and the lambda timeout needs to be longer for the file format because of how slow the file format is.

Reply:
I'm wary of introducing more dependencies between things as this causes problems when creating and destroying a new environment. 
Also, because there's one file for each lambda, we'd need an output in the lambda module for each lambda timeout like this:
```
output "file_format_timeout" {
  value = aws_lambda_function.file_format_lambda_function.timeout
}

output "checksum_timeout" {
  value = aws_lambda_function.checksum_lambda_function.timeout
}

output "download_files_timeout" {
  value = aws_lambda_function.download_files_lambda_function.timeout
}
... All other lambdas with associated queues
```
That's introducing a lot of output variables. In general it's quite a few changes to avoid something that is easy enough to explain. I could put a comment above this if it helps?

Reply:
OK, in that case a comment sounds good. Just trying to make it easier for people to change things without either accidentally breaking things or worrying about accidentally breaking things. 👍